src/include/sof/audio/module_adapter/module/module_interface.h

```diff
@@ -67,6 +67,8 @@ struct output_stream_buffer {
 	uint32_t size; /* size of data in the buffer */
 };
 
+struct comp_dev;
+struct timestamp_data;
```

Comment:
actually this file also should include stdint.h and stdbool.h

Reply:
can be fixed incrementally.